SPECS/azurelinux-repos/azurelinux-repos.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        AzureLinux repo files, gpg keys
 Name:           azurelinux-repos
-Version:        3.0
+Version:        %{azl}.0
```

Comment:
double checking here:
Does our `autoupgrade` tooling work with this change ?

Reply:
I don't think so, but I believe the auto-upgrades are for cases where we can get a newer spec from another distro. The specs modified in this PR are the ones we either created ourselves or they moved away from the original source so much (for instance due to a name change), that an auto-upgrade wouldn't work anyway.